index.js

```diff
@@ -228,9 +229,11 @@ There is a typo in your dataset 😱 The 9th artist, Vincent Van Gogh is current
  
  Example, if getArtistByIndex is invoked with the artists array and the number 0, it will return `the artist at index 0 is Amedeo Modigliani` */
 
-function getArtistByIndex(/*Your Code Here*/) {
-  /*Your Code Here*/
+function getArtistByIndex(array, number) {
+  const name = artists[number].name;
```

Comment:
Line 233 hard codes the array -----> const name = artists[number].name;  

You will want to make use of the array parameter passed into your function like so ——> const name = array[number].name;